docker/README.md

```diff
@@ -0,0 +1,81 @@
+# Run GreenGuard using Docker
+
+GreenGuard is prepared to be run using [Docker](https://docker.com/).
+
+This are the commands needed to start a Docker container locally that runs a [Jupyter Notebook](
+https://jupyter.org/) already configured to run GreenGuard.
+
+```bash
+docker run -ti -p8888:8888 signals-dev/greenguard:0.2.2.dev0
+```
+
+Further details about the usage of this image can be found [here](
+https://hub.docker.com/repository/docker/signalsdev/greenguard).
+
+## Run GreenGuard on Kubernetes
+
+GreenGuard can also be started using [Kubernetes](https://kubernetes.io/).
+
+Here are the minimum steps to do so:
+
+1. Create a POD yaml file with the these contents:
```

Comment:
1. Create a yaml file with these contents:

For this example, we are assuming that the yaml file is named `greenguard-pod.yml`.